src/boards/p8/board.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ * Copyright (C) 2020 Daniel Thompson
+ */
+
+#ifndef RELOADER_BOARD_P8_H__
+#define RELOADER_BOARD_P8_H__
+
+#define CONFIG_ST7789
+
+// WDT config
+#define CONFIG_HAVE_WDT_BUTTON   (1)
+#define CONFIG_WDT_BUTTON        (17)
+#define CONFIG_WDT_BUTTON_ACTIVE (1) // active high
+#define CONFIG_WDT_BUTTON_PULL   (0)  // no pull up/down
+//#define CONFIG_WDT_BUTTON_ENABLE (15)
```

Comment:
Let's drop the commented out line.

Reply:
(ah yes... and can you add a Signed-off-by: too)

Reply:
- Commented out line dropped
- Added Signed-off-by commit message